modules/odata/odata-core/src/main/java/org/eclipse/dirigible/engine/odata2/sql/binding/EdmTableBinding.java

```diff
@@ -217,6 +217,14 @@ public ColumnInfo getColumnInfo(EdmProperty property) {
         }
     }
 
+    public List<String> getParameters() throws EdmException {
+        return readMandatoryConfig("_parameters_", List.class);
+    }
+
+    public String getDataStructureType() throws EdmException {
+        return readMandatoryConfig("dataStructureType", String.class);
+    }

```

Comment:
Let's make them not mandatory but optional with default values in case there are missing. Otherwise we will break current functionality and working applications.